site2/docs/reference-configuration.md

```diff
@@ -129,7 +129,7 @@ BookKeeper is a replicated log storage system that Pulsar uses for durable stora
 |dbStorage_rocksDB_numLevels||-1|
 |dbStorage_rocksDB_numFilesInLevel0||4|
 |dbStorage_rocksDB_maxSizeInLevel1MB||256|
-
+bookieId | If you want to custom bookie ID or use a dynamic network address for the bookie, you can set this option. <br><br>Bookie advertises itself using `bookieId` rather than `BookieSocketAddress` (hostname:port or IP:port).<br><br> `bookieId` is a non empty string that can contain ASCII digits and letters ([a-zA-Z9-0]), colons, dashes, and dots. <br><br>For more information about `bookieId`, see [here](http://bookkeeper.apache.org/bps/BP-41-bookieid/).|/|
```

Comment:
probably it is better to move this line next to `useHostNameAsBookieID`
Also probably it is worth telling that if you set a `bookieId` the `useHostNameAsBookieID` does not have effect

Reply:
Also I see that the default value that I see in the rendered table is  '/', we should leave an empty string

Reply:
@eolivelli thanks for your comments, I've incorporated, PTAL